packages/web-components/src/checkbox/checkbox.styles.ts

```diff
@@ -67,7 +67,7 @@ export const styles = css`
     font-size: ${fontSizeBase300};
     line-height: ${lineHeightBase300};
     color: ${colorNeutralForeground3};
-    padding: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
+    padding-inline: ${spacingVerticalS} ${spacingHorizontalS} ${spacingVerticalS} ${spacingHorizontalXS};
```

Comment:
you can only pass `padding-inline` the start and end values, you will still need padding for the top and bottom values

https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline